contracts/BSCValidatorSet.sol

```diff
@@ -97,6 +118,8 @@ contract BSCValidatorSet is IBSCValidatorSet, System, IParamSubscriber, IApplica
   event unexpectedPackage(uint8 channelId, bytes msgBytes);
   event paramChange(string key, bytes value);
   event feeBurned(uint256 amount);
+  event validatorEnterMaintenance(address indexed validator);
+  event validatorExitMaintenance(address indexed validator);
 
   /*********************** init **************************/
   function init() external onlyNotInit{
```

Comment:
Please operate `validatorExtraSet` in init func too.

Reply:
fixed